cupy/cuda/compiler.py

```diff
@@ -115,30 +117,10 @@ def _get_nvrtc_version():
     return _nvrtc_version
 
 
-# Known archs for Tegra/Jetson/Xavier/etc
-_tegra_archs = ('53', '62', '72')
-
-
 @_util.memoize(for_each_device=True)
 def _get_arch():
-    # See Supported Compile Options section of NVRTC User Guide for
-    # the maximum value allowed for `--gpu-architecture`.
-    major, minor = _get_nvrtc_version()
-    if major < 10 or (major == 10 and minor == 0):
-        # CUDA 9.x / 10.0
-        _nvrtc_max_compute_capability = '70'
-    elif major < 11:
-        # CUDA 10.1 / 10.2
-        _nvrtc_max_compute_capability = '75'
-    else:
-        # CUDA 11.0 / 11.1
-        _nvrtc_max_compute_capability = '80'
-
     arch = device.Device().compute_capability
-    if arch in _tegra_archs:
-        return arch
-    else:
-        return min(arch, _nvrtc_max_compute_capability)
+    return arch
```

Comment:
I simplify this function by just returning the current device's CC. The current implementation was initially done by @kmaehashi in #1055 due to the NVRTC issue in CUDA 8. But since we drop CUDA 8 support, and apparently the issue is resolved in CUDA 9+ (all CCs' supported by CUDA are listed in the NVRTC option), looks like we no longer need this workaround.

Refs:
- https://docs.nvidia.com/cuda/archive/9.2/nvrtc/index.html#group__options (CUDA 9.2)
- https://docs.nvidia.com/cuda/archive/10.0/nvrtc/index.html#group__options (CUDA 10.0)
- https://docs.nvidia.com/cuda/archive/10.1/nvrtc/index.html#group__options (CUDA 10.1)
- https://docs.nvidia.com/cuda/archive/10.2/nvrtc/index.html#group__options (CUDA 10.2)
- https://docs.nvidia.com/cuda/archive/11.0/nvrtc/index.html#group__options (CUDA 11.0)
- https://docs.nvidia.com/cuda/archive/11.1.1/nvrtc/index.html#group__options (CUDA 11.1.1)